### DISCUSSION TITLE
Using the current cell's value in :fill?

### DISCUSSION THREAD

**Discussion by 097115** [andmarti1424/sc-im#863] [Q&A]
Can I somehow `:fill` a range using the current cell's value as the initial number without entering it manually or copying/pasting? 

Referencing cells as with functions doesn't seem to work with commands, right?

**Comment by fg-charles** [comment]
You could write a Lua script for that. Using something like `sc.lgetnum` followed by `sc.lsetform`. See the help for more info on Lua scripts.
